<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_8">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_9">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_10">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_11">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>INPUT MATRIX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>GENERATE 2D LIST</string>
    </property>
   </widget>
   <widget class="Line" name="line_10">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_12">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line_11">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>150</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_12">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_13">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_14">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_13">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line_15">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_16">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_17">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_14">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line_19">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_20">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_18">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_15">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line_21">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_22">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_24">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>160</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_16">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line_25">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_26">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_27">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_17">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line_29">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_30">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>90</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_18">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line_33">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>150</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_34">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>150</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2D LINKED LIST</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Pos_00">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Pos_12">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Pos_11">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Pos_10">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Pos_02">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Pos_01">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Pos_21">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Pos_20">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Pos_22">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NPos_00">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NPos_01">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NPos_22">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NPos_21">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NPos_20">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NPos_12">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NPos_11">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NPos_10">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NPos_02">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="perimeter_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Perimeter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="diagonalSum_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Diagonal Sum</string>
    </property>
   </widget>
   <widget class="QPushButton" name="transpose_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Transpose</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="perimeter">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>200</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="diagonalSum">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="transpose">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>280</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>NONE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <zorder>tableView_4</zorder>
   <zorder>tableView_13</zorder>
   <zorder>tableView_14</zorder>
   <zorder>tableView_15</zorder>
   <zorder>tableView_5</zorder>
   <zorder>tableView_17</zorder>
   <zorder>tableView_18</zorder>
   <zorder>tableView_16</zorder>
   <zorder>tableView_12</zorder>
   <zorder>tableView</zorder>
   <zorder>tableView_2</zorder>
   <zorder>tableView_3</zorder>
   <zorder>tableView_6</zorder>
   <zorder>tableView_7</zorder>
   <zorder>tableView_8</zorder>
   <zorder>tableView_9</zorder>
   <zorder>tableView_10</zorder>
   <zorder>tableView_11</zorder>
   <zorder>label</zorder>
   <zorder>pushButton</zorder>
   <zorder>line_10</zorder>
   <zorder>line_11</zorder>
   <zorder>line_12</zorder>
   <zorder>line_5</zorder>
   <zorder>line_6</zorder>
   <zorder>line_7</zorder>
   <zorder>line_8</zorder>
   <zorder>line_13</zorder>
   <zorder>line_14</zorder>
   <zorder>line_15</zorder>
   <zorder>line_16</zorder>
   <zorder>line_17</zorder>
   <zorder>line_19</zorder>
   <zorder>line_20</zorder>
   <zorder>line_9</zorder>
   <zorder>line_18</zorder>
   <zorder>line_21</zorder>
   <zorder>line_22</zorder>
   <zorder>line_24</zorder>
   <zorder>line_25</zorder>
   <zorder>line_26</zorder>
   <zorder>line_27</zorder>
   <zorder>line_29</zorder>
   <zorder>line_30</zorder>
   <zorder>line_33</zorder>
   <zorder>line_34</zorder>
   <zorder>line_3</zorder>
   <zorder>line_4</zorder>
   <zorder>line_2</zorder>
   <zorder>line</zorder>
   <zorder>label_2</zorder>
   <zorder>Pos_00</zorder>
   <zorder>Pos_12</zorder>
   <zorder>Pos_11</zorder>
   <zorder>Pos_10</zorder>
   <zorder>Pos_02</zorder>
   <zorder>Pos_01</zorder>
   <zorder>Pos_21</zorder>
   <zorder>Pos_20</zorder>
   <zorder>Pos_22</zorder>
   <zorder>NPos_00</zorder>
   <zorder>NPos_01</zorder>
   <zorder>NPos_22</zorder>
   <zorder>NPos_21</zorder>
   <zorder>NPos_20</zorder>
   <zorder>NPos_12</zorder>
   <zorder>NPos_11</zorder>
   <zorder>NPos_10</zorder>
   <zorder>NPos_02</zorder>
   <zorder>perimeter_2</zorder>
   <zorder>diagonalSum_2</zorder>
   <zorder>transpose_2</zorder>
   <zorder>perimeter</zorder>
   <zorder>diagonalSum</zorder>
   <zorder>transpose</zorder>
   <zorder>pushButton_2</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>552</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
